### ISSUE TITLE
scroll day by day in week view (no jumping viewport)

### ISSUE THREAD

**Issue by alexanderdd** [FossifyOrg/Calendar#218]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Calendar/issues) or [closed](https://github.com/FossifyOrg/Calendar/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Calendar/discussions) or [closed](https://github.com/FossifyOrg/Calendar/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Calendar/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

To reproduce:
1. select week view, set to 7 days
2. scroll to the right
3. the viewport jumps back to either the current week or the next week as soon as you stop touching

Expected behaviour:
The viewport should stay at the days I have scrolled to, no jumping.

Comments:
If I want to look at the days I scrolled to, the UI should let me, not force some other view. I use 3day view a lot, and the current behaviour is bad UX, I cant look at the days I want to, viewport always jumps to other days.

### Why do you want this feature?

Better UX

**Comment by alexanderdd** [comment]
@Aga-C can you triage please?

**Comment by acress1** [comment]
Hi
Is it expected to be dev?
Thx

**Comment by naveensingh** [comment]
I don't think this is worth implementing. The weekly view is intentionally paged to focus on a stable window of time, not a rolling one, and many other calendar apps behave the same way. There are existing options **Start week on** and **Start week with current day** that already cover use cases such as "always show me the next 5 days".
